3 - Assignment/240326 - Rust Basic + Roguelike Game, Week 1/fibonacci/src/lib.rs

```diff
@@ -1,13 +1,23 @@
 pub fn fib(n: u32) -> u32 {
     if n <= 2 {
-        // The base case
-        todo!("Implement this");
+        1
     } else {
         // The recursive case
-        todo!("Implement this");
+        return fib(n-1) + fib(n-2);
```

Comment:
`return`을 생략할 수 있습니다.
```suggestion
        fib(n - 1) + fib(n - 2)
```